tests/core/test_aws.py

```diff
@@ -70,3 +73,63 @@ def test_force_delete_terraform_lock_id(
             ],
             Key={"LockID": {"S": f"{aws_layer.state_storage()}/{aws_layer.name}"}},
         )
+
+    def test_get_detailed_config_map_configuration_present(
```

Comment:
also rename the tests accordingly `test_new _name`
otherwise if the test fail someone will not know what is broken from the name